client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -359,6 +360,8 @@ function CheckoutSummaryFeaturesList( props: {
 		isDomainTransfer( product )
 	);
 
+	usePresalesChat( 'wpcom', hasDomainTransferProduct );
```

Comment:
From this comment (p1690884716213019/1690848330.283159-slack-C05CT832K2T), I now remember seeing the presales chat bubble on the checkout page before, and wondering if this is still needed.